app/Core/Database.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Core;
-
+$config_data = parse_ini_file("dbconfig.ini");
```

Comment:
This should also be a part of the initialize method.

Reply:
Moved